### PR TITLE
Coding practice11

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/SortArray.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package codingPractice;
+
+public class SortArray {
+    public static void main(String[] args) {
```
Comment (Ams12) [inline simple/src/codingPractice/SortArray.java:4]: There should be space in between class and method.

File: simple/src/codingPractice/SortArray_ReadMe [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Java Program to sort the elements of an array in ascending order
+"C:\Program Files\Java\jdk-17.0.4.1\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\lib\idea_rt.jar=61065:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2022.3.3\bin" -Dfile.encoding=UTF-8 -classpath C:\Users\suyogw\Desktop\Auto\NewMyWayRepo\simple\out\production\simple codingPractice.SortArray
+Elements of original array:
+5 2 8 7 1  Elements of array sorted in ascending order:
```
Comment (Ams12) [inline simple/src/codingPractice/SortArray_ReadMe:4]: Output(Elements of array sorted in ascending order) should be in the next line.

**Review by Ams12 — CHANGES_REQUESTED** [review]

File: simple/src/codingPractice/SortArray_ReadMe (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/SortArray_ReadMe:3]: done

File: simple/src/codingPractice/SortArray.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/SortArray.java:4]: done

**Review by Ams12 — APPROVED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**